core/src/main/java/org/pentaho/di/core/bowl/HasBowlInterface.java

```diff
@@ -0,0 +1,37 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2024 by Hitachi Vantara : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.di.core.bowl;
+
+public interface HasBowlInterface {
```

Comment:
could be just HasBowl?

Reply:
All the other similar interfaces used by AbstractMeta use "Interface". There are 8 others already.

Reply:
But does it mean anything? Looking at AbstractMeta, seems like interfaces created by matt casters end in Interface, and those created by sven boden don't. Doesn't seem to be any intentional pattern in the rest of the code either, where those ending with Interface are below half of total